### PR TITLE
extended widom insertion to allow the user to determine multiple exce…

### PR DESCRIPTION
**Pull request by jonaslandsgesell**
…ss chemical potentials at once

### CODE REVIEW

File: src/core/reaction_ensemble.cpp [hunk 1]
```diff
@@ -1765,6 +1765,15 @@ double ConstantpHEnsemble::calculate_acceptance_probability(
   return bf;
 }
 
+void WidomInsertion::add_reaction (double Gamma, const std::vector<int> &_reactant_types,
```
Comment (fweik) [inline src/core/reaction_ensemble.cpp]: Please stick to the naming convention.

**Review by fweik — CHANGES_REQUESTED** [review]

File: src/core/reaction_ensemble.cpp (hunk 1 above)
Comment (jonaslandsgesell) [reply src/core/reaction_ensemble.cpp]: Which one? https://github.com/espressomd/espresso/wiki/Naming-conventions-and-folder-structure I do not see which convention listed there would apply.

Comment (jonaslandsgesell) [reply src/core/reaction_ensemble.cpp]: Seems like you refer to https://stackoverflow.com/questions/228783/what-are-the-rules-about-using-an-underscore-in-a-c-identifier

**Comment by fweik** [comment]
You also have to rename `Gamma` to `gamma`.

**Review by fweik — APPROVED** [review]

**Comment by jonaslandsgesell** [comment]
Also fixes #1981.

**Pull request merged**